src/main/java/com/example/TicketBookingApp/service/ScreeningService.java

```diff
@@ -0,0 +1,64 @@
+package com.example.TicketBookingApp.service;
+
+import com.example.TicketBookingApp.domain.Screening;
+import com.example.TicketBookingApp.domain.dto.ScreeningDetailsResponse;
+import com.example.TicketBookingApp.domain.dto.ScreeningResponse;
+import com.example.TicketBookingApp.exception.ScreeningNotFoundException;
+import com.example.TicketBookingApp.repository.ScreeningRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class ScreeningService {
+
+    private final ScreeningRepository screeningRepository;
+
+    public List<ScreeningResponse> getScreeningByDate(LocalDate date, LocalTime start, LocalTime end) {
+        List<Screening> screenings = screeningRepository.findByDateAndStartTimeBetween(date, start, end);
+        List<ScreeningResponse> screeningResponses;
+
+        screeningResponses = screenings
+                .stream()
+                .map(this::mapScreeningToScreeningResponse)
+                .collect(Collectors.toList());
+
+        return screeningResponses
+                .stream()
+                .sorted(Comparator.comparing(ScreeningResponse::getTitle)
```

Comment:
dlaczego baza danych nie sortuje?

Reply:
Jak to się zachowa dla polskich znaków?

Reply:
Było to celowe działanie z mojej strony w celu przedstawienia obsługi strumieni. Wiem, że wystarczyłaby odpowiednia adnotacja nad nazwą pola. Odpowiadając na drugie pytanie, niestety polskich znaków nie weźmie pod uwagę przy sortowaniu.